rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/ValidationApiFailureAcceptance.java

```diff
@@ -0,0 +1,148 @@
+package gov.cms.qpp.conversion.api.acceptance;
+
+import com.github.mustachejava.DefaultMustacheFactory;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+import gov.cms.qpp.conversion.model.error.AllErrors;
+import gov.cms.qpp.conversion.model.error.Detail;
+import gov.cms.qpp.conversion.xml.XmlException;
+import gov.cms.qpp.conversion.xml.XmlUtils;
+import gov.cms.qpp.test.annotations.AcceptanceTest;
+import io.restassured.response.Response;
+import org.jdom2.Attribute;
+import org.jdom2.filter.Filter;
+import org.jdom2.filter.Filters;
+import org.jdom2.xpath.XPathExpression;
+import org.jdom2.xpath.XPathFactory;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.http.HttpStatus;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import static com.google.common.truth.Truth.assertThat;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.fail;
+
+@ExtendWith(RestExtension.class)
+class ValidationApiFailureAcceptance {
+	private static final XPathFactory XPF = XPathFactory.instance();
+	private MustacheFactory mf = new DefaultMustacheFactory();
+	private Mustache fixture = mf.compile("valid-QRDA-III-latest-fixture.xml");
+
+
+	@AcceptanceTest
+	void testSuccess() {
+		Response response = performRequest(getQrda(null));
+
+		assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED.value());
+	}
+
+	@ParameterizedTest
+	@Tag("acceptance")
```

Comment:
Let's add a meta annotation @Acceptance, and annotate @AcceptanceTest with it

Reply:
We already have an annotation `AcceptanceTest`, which is used above.  Is the reason that annotation isn't used here is because the `@Test` of `AcceptanceTest` interferes with the `@ParameterizedTest` here?  If so, perhaps make an `AcceptanceParameterizedTest` annotation?